packages/frontend/vite.config.ts

```diff
@@ -1,4 +1,5 @@
-import { defineConfig } from "vitest/config";
+/// <reference types="vitest" />
```

Comment:
What is the reason for this typescript directive?

Reply:
Not entirely sure but it's recommended here: https://vitest.dev/config/

Reply:
I think it can be removed since the types are already in the tsconfig file